pkg/controller/ground.go

```diff
@@ -675,3 +704,21 @@ func (op *GroundControl) podUpdate(cur, old interface{}) {
 		}
 	}
 }
+
+func (op *GroundControl) ensureFinalizerCreated(kluster *v1.Kluster) (err error) {
```

Comment:
Maybe we should move this to a util function? This will be useful in every controller and its already duplicated in this PR for `launchctl`

Reply:
done